composer.json

```diff
@@ -45,10 +45,15 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "scripts": {
-        "cs-check": "vendor/bin/phpcs",
-        "cs-fix": "vendor/bin/phpcbf",
-        "stan": "vendor/bin/phpstan analyse",
-        "test": "vendor/bin/phpunit"
+        "cs-check": "phpcs",
+        "cs-fix": "phpcbf",
+        "stan": "phpstan analyze src/",
```

Comment:
we have phpstan.neon configuration file, where we have the same path, as result we might skip it and use only
`"stan": "phpstan",`

Reply:
thanks, you are right of course, I missed it.